packages/restic/build.sh

```diff
@@ -0,0 +1,31 @@
+TERMUX_PKG_MAINTAINER="Leonid Plyushch <leonid.plyushch@gmail.com> @xeffyr"
+
+TERMUX_PKG_HOMEPAGE=https://restic.net/
+TERMUX_PKG_DESCRIPTION="Fast, secure, efficient backup program"
+TERMUX_PKG_VERSION=0.9.2
+TERMUX_PKG_SRCURL=https://github.com/restic/restic/archive/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=8f8eee3e9651b9f7384a323ba3c26a5667a6388ab2ef8e6d869d3cd69b9f7c95
+#TERMUX_PKG_RECOMMENDS="dropbear | openssh, rclone"
```

Comment:
Either recommend or don't, no point in a comment.

Reply:
Thanks.